src/app/course/course.component.html

```diff
@@ -6,6 +6,10 @@
         {{ course?.name }}
     </h1>
 
+    <h3>
```

Comment:
This should have proper class names for Taiga UI headers.

Reply:
For example, when we enable dark mode this text is probably gonna stay black without the proper class names and therefore, it will be hidden in the dark mode.

Reply:
Ok, I updated this.